sql/core/benchmarks/CollationBenchmark-results.txt

```diff
@@ -2,26 +2,26 @@ OpenJDK 64-Bit Server VM 17.0.10+7-LTS on Linux 6.5.0-1016-azure
 AMD EPYC 7763 64-Core Processor
 collation unit benchmarks - equalsFunction:  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 --------------------------------------------------------------------------------------------------------------------------
-UTF8_BINARY_LCASE                                   34122          34152          42          0.0      341224.2       1.0X
-UNICODE                                              4520           4522           2          0.0       45201.8       7.5X
-UTF8_BINARY                                          4524           4526           2          0.0       45243.0       7.5X
-UNICODE_CI                                          52706          52711           7          0.0      527056.1       0.6X
+UTF8_BINARY_LCASE                                    8006           8022          24          0.0       80056.6       1.0X
```

Comment:
Can you add a benchmark for UTF8_BINARY_LCASE with non ASCII chars?
This should be a separate group in benchmark list.

Reply:
Added new benchmarks for ASCII and non-ASCII data, please check.